Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -959,11 +982,15 @@ def __init__(self, readerClass=None, writerClass=None, fontClass=None):
             self.writerClass = writerClass
         else:
             self.writerClass = BaseDocWriter
-        if fontClass is not None:
-            self.fontClass = fontClass
-        else:
-            from defcon.objects.font import Font
-            self.fontClass = Font
+        self._fontClass = fontClass
+
+    @property
+    def fontClass(self):
+        if self._fontClass is not None:
+            return self._fontClass
+
+        from defcon.objects.font import Font
+        return Font
```

Comment:
Here I import `defcon` only when someone calls `fontClass` or its only caller, `getFonts()`

Reply:
and who is calling getFonts()? Can't clients re-implement their own getFonts()? I think DesignSpaceDocument can live without a self.fontClass property, at least until we have a working fontTools.ufoLib.Font implementation to use as default.